src/ssl.c

```diff
@@ -44954,7 +45072,7 @@ void wolfSSL_sk_X509_NAME_free(WOLF_STACK_OF(WOLFSSL_X509_NAME)* sk)
 
     XFREE(sk, sk->heap, DYNAMIC_TYPE_OPENSSL);
 }
-
+#if 1 /*------------------------test ------------------------------*/
```

Comment:
Looks like a test/debug line got left in here?  Please remove.

Reply:
Removed the line.